### PR TITLE
Fix pom

### PR DESCRIPTION
**Pull request by AlexanderNikitin-Smartbear**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -15,15 +15,15 @@
    See the License for the specific language governing permissions and
    limitations under the License.
 -->
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-4.0.0.xsd">
```
Comment (antone-sb) [inline pom.xml:18]: Why did you change schema location?

Comment (AlexanderNikitin-Smartbear) [reply pom.xml:18]: To comfortable editing the file in Idea. The old address redirects to a new address, but Idea does not understand it, and the file is completely red.
